static/bidder-params/seedingAlliance.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Seeding Alliance Adapter Params",
+    "description": "A schema which validates params accepted by the Seeding Alliance adapter",
+    "type": "object",
+    "properties": {
+        "adUnitId": {
+            "type": "string",
+            "description": "Ad Unit ID"
+        }
+    }
+}
```

Comment:
Just to confirm: adUnitId is not a required parameter and can be omitted, right?

Reply:
Thanks for noticing that.  Should have been actually set as "required".